scss/components/_icons.scss

```diff
@@ -113,3 +113,18 @@ h6 {
     }
   }
 }
+@media screen and (-ms-high-contrast: active) {
+  .rounded-circle {
+    -ms-high-contrast-adjust: none;
+
+    &.bg-primary {
+      background-color: HighLight !important;
+    }
+    &.bg-300 {
```

Comment:
`.bg-300` isn't a semantically meaningful selector.  It doesn't communicate why we're overriding the background color.

What else can we use here?